File: NOVEMBER-2025/09-11-2025/p2169.php
class Solution {
    /**
     * @param Integer $num1
     * @param Integer $num2
     * @return Integer
     */
    function countOperations($num1, $num2) {
        $counter = 0;
        while($num1 && $num2){
            ($num1>=$num2) ? $num1 = $num1 - $num2 : $num2 = $num2-$num1;
            $counter++;    
        }
    return $counter;
    }
}

File: NOVEMBER-2025/09-11-2025/p2169.rs
impl Solution {
    pub fn count_operations(mut num1: i32, mut num2: i32) -> i32 {
        let mut o = 0;
        while num1!=0 && num2!= 0 {
            if num1>=num2{
                num1-=num2
            }else{
                num2-=num1
            }
            o+=1;
        }

        return o;
    }
}